insta-demo/Vagrantfile

```diff
@@ -28,4 +28,10 @@ Vagrant.configure("2") do |config|
     a.vm.provision "shell",
       path: "pulp-insta-demo.sh"
   end
+
+  config.vm.define "bionic-pulp-insta-demo" do |a|
+    a.vm.box = "generic/ubuntu1804"
```

Comment:
I've had problems with these boxes in the past, they do some weird shit with ipv6 that messes with redis.  Not sure if it's changed in the past year.

Reply:
Namely, I found that they disabled ipv6, and there's a pre-or-post-install script in the redis package that tried to ping redis on ipv6, and the package installation process totally locked up until eventually failing.

Reply:
I tested it and it Vagrant still provisions successfully.

The reason why it's not a problem is that IPv6 is enabled at the kernel level still, and containers via network namespaces handle the rest of the IPv6 stack, abstracting away any Ubuntu 18.04 userspace/config issues. I researched this per:
https://pulp.plan.io/issues/5414